addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComChimeMessage.java

```diff
@@ -0,0 +1,235 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rfxcom.internal.messages;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.smarthome.core.library.items.NumberItem;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.Type;
+import org.eclipse.smarthome.core.types.UnDefType;
+import org.openhab.binding.rfxcom.RFXComValueSelector;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComException;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComUnsupportedValueException;
+
+/**
+ * RFXCOM data class for chime messages.
+ *
+ * @author Mike Jagdis
+ */
+public class RFXComChimeMessage extends RFXComBaseMessage {
+
+    public enum SubType {
+        BYRONSX(0),
+        BYRONMP001(1),
+        SELECTPLUS(2),
+        SELECTPLUS3(3),
+        ENVIVO(4),
+
+        UNKNOWN(255);
+
+        private final int subType;
+
+        SubType(int subType) {
+            this.subType = subType;
+        }
+
+        public byte toByte() {
+            return (byte) subType;
+        }
+
+        public static SubType fromByte(int input) throws RFXComUnsupportedValueException {
+            for (SubType c : SubType.values()) {
+                if (c.subType == input) {
+                    return c;
+                }
+            }
+
+            throw new RFXComUnsupportedValueException(SubType.class, input);
+        }
+    }
+
+    private static final List<RFXComValueSelector> SUPPORTED_INPUT_VALUE_SELECTORS = Arrays
+            .asList(RFXComValueSelector.SIGNAL_LEVEL, RFXComValueSelector.CHIME_SOUND);
+
+    private static final List<RFXComValueSelector> SUPPORTED_OUTPUT_VALUE_SELECTORS = Collections
+            .singletonList(RFXComValueSelector.CHIME_SOUND);
+
+    public SubType subType = SubType.UNKNOWN;
+    public int sensorId = 0;
+    public int chimeSound = 0;
+    public byte signalLevel = 0;
+
+    public RFXComChimeMessage() {
+        packetType = PacketType.CHIME;
+    }
+
+    public RFXComChimeMessage(byte[] data) throws RFXComException {
+        encodeMessage(data);
+    }
+
+    @Override
+    public String toString() {
+        String str = "";

```

Comment:
Maybe assign the `super.toString()` here, prepending this empty string feels rather useless. I know its also the case for the other bindings I have a passive branch for that :-) So I can just take it along at that time.